proto-lens/package.yaml

```diff
@@ -4,11 +4,11 @@ synopsis: A lens-based implementation of protocol buffers in Haskell.
 description: >
   The proto-lens library provides to protocol buffers using modern
   Haskell language and library patterns.  Specifically, it provides:
-   
+
```

Comment:
Don't remove this whitespace; it messes up the comments due to interactions between Haddock/cabal/hpack.

Reply:
that's autosave :(